system/security/shared/caching.cpp

```diff
@@ -723,43 +709,48 @@ void CPermissionsCache::managedFileScopesCacheFillThread()
         time_t now;
         time(&now);
 
-        // Save previous fill time in case another thread modifies it
-        time_t previousCacheFillTime = m_lastCacheFillTime;
+        time_t copyLastCacheFillTime = m_lastCacheFillTime;  // atomic read
```

Comment:
trivial: explicitly using load() would make it clear it's an atomic read without comment